README.md

```diff
@@ -8,6 +8,10 @@ GET `/api/v1/health`
 ### List locations
```

Comment:
@iloveicedgreentea One of the FEMA folks called my attention to the fact that the document lacks any explicit mention of the base URL. I recognize that a savvy developer could figure it out, but it seems that not having it on the readme is a hurdle for some of the folks interested in our data

Reply:
Updated readme with base url